mobile/mobile.go

```diff
@@ -32,32 +32,20 @@ type Callback interface {
 	Call(data []byte, err error)
 }
 
-// NewWallet creates a brand new wallet and returns its recovery phrase
-func NewWallet(wordCount int) (string, error) {
-	wcount, err := wallet.NewWordCount(wordCount)
-	if err != nil {
-		return "", err
-	}
-
-	w, err := wallet.NewWallet(wcount.EntropySize())
-	if err != nil {
-		return "", err
-	}
-
-	return w.RecoveryPhrase, nil
+func MarshalWalletAccount(account *keypair.Full) ([]byte, error) {
```

Comment:
You still need a` gobind` compatible method for creating a wallet... `keypair.Full` won't make it across the "bridge" and even so, the user isn't able to make on directly. https://godoc.org/golang.org/x/mobile/cmd/gobind#hdr-Type_restrictions. How about a mobile specific `WalletFromWordCount`?

Reply:
If you're keeping `MarshalWalletAccount` as a utility, make it private (lowercase), so as to not clutter the `gobind`ed API.